clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/Executor.scala

```diff
@@ -15,8 +15,19 @@ trait Executor
     with FailFastCirceSupport
     with ModelAutoDerivation
     with FutureWithErrorMessage {
+
   def execute(webClient: ClioWebClient, ioUtil: IoUtil)(
     implicit ec: ExecutionContext,
     credentials: HttpCredentials
   ): Future[HttpResponse]
+
+  protected def deleteGoogleObject(path: String, ioUtil: IoUtil): Future[Unit] = {
+    ioUtil.deleteGoogleObject(path) match {
+      case 0 => Future.successful(())
+      case _ =>
+        Future.failed(
```

Comment:
It seems like we should bake error handling into `IoUtil` itself instead of making the caller do it. We should also trap the command output and put it in the exception (in this case, since it's run from clio-client, it will all get dumped to stdout, but that seems like a hack to me). Or maybe this can wait until we use the google filesystem library?